src/main/java/com/ericsson/eiffel/remrem/publish/service/MessageServiceRMQImpl.java

```diff
@@ -168,4 +208,130 @@ private void createFailureResult(List<PublishResultItem> events) {
                 PropertiesConfig.INVALID_EVENT_CONTENT);
         events.add(event);
     }
+    
+    
+    /**
+     * This method handles multiple events status codes
+     * @param routingKeyMap contains the eventId and routing key of that event
+     * @param msgService Messaging service
+     * @param json
```

Comment:
minor note. All public methods (and why not private methods) should have a good description of the parameters. Stating "json"  doesn't give me any information about what this parameter is used for. Look at any definition in the java standard libraries for inspiration: example: https://docs.oracle.com/javase/6/docs/api/java/lang/String.html#indexOf(java.lang.String, int)